test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1564,9 +1574,16 @@ def tearDown(self):
 
         for sched in self.scheds:
             self.scheds[sched].cleanup_files()
+        self.revert_server(self.server)
+        self.revert_moms()
+        self.revert_pbsconf()
+        self.server.load_configuration(self.saved_file)
+        self.du.rm(path=self.saved_file)
 
         self.log_end_teardown()
 
     @classmethod
     def tearDownClass(cls):
         cls._testMethodName = 'tearDownClass'
+        cls.server.load_configuration(cls.saved_file)
```

Comment:
why are you loading configuration in tearDownClass()?

Reply:
@agrawalravi90 as per the EDD: https://pbspro.atlassian.net/wiki/spaces/PD/pages/557088789/Design+for+a+supported+way+to+change+default+setup+in+PTL
It says to load PBS everytime in teardown and teardown class. 
Goal is to revert pbs to the same state after running the tests. Even when --use-current-setup is not used